source/docs/magento1/add-custom-endpoint.md

```diff
@@ -0,0 +1,251 @@
+---
+id: add-custom-endpoint
+title: Add your custom endpoint
+---
+
+When customizing your shop, you may need to retrieve data from Magento. This is done by fetching data from an REST endpoint on Magento that will be used in a [GraphQL module](/docs/essentials/extend-the-graphql-schema.html). In this documentation, you will learn how to create a new REST endpoint in your Magento 1.
+
+This can be done by completing the following steps:
+
+1. Add and complete `api2.xml` file
+2. Implement the API method(s)
+
+## Add and complete `api2.xml`
+
+`api2.xml` is your main API config file, you can add this file on your local module in `etc` directory.
```

Comment:
I tend to add a comment at the beginning of each code example with the full path. However, when I do so it is more likely to be a guide rather than an API reference. Not sure in which case we are here. It's mid way between a guide and a reference page.

If we are adding full module paths (which is better imo), we should add also context at the beginning of the page explaining that for the examples shown below, we are creating an endpoint that manages an example resource (with a concrete example).

Reply:
I use module name on the example step, here this is for explain the different node.